system/ee/ExpressionEngine/Library/CP/EntryManager/ColumnIconDate.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace ExpressionEngine\Library\CP\EntryManager;
+
+use ExpressionEngine\Library\CP\EntryManager\Columns\Column;
+
+abstract class ColumnIconDate extends Column
```

Comment:
Maybe more generic `DateColumn` instead of `ColumnIconDate` would work better?

Reply:
Agreed. Will update.